grpc/flatbuffers-java-grpc/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
rename this directory from `flatbuffers-java-grpc` to just `java` ? The other two parts are obvious from context